src/resources/locales/en.json

```diff
@@ -163,7 +166,7 @@
         "title": "Settings",
         "theme": {
           "title": "Theme",
-          "predefined": "Predefined themes",
+          "predefined": "Predifined themes",
```

Comment:
The correct spelling is `Predefined`

Reply:
you're right, I didn't request this. @manuellagrand ?